peakdet/cli/run.py

```diff
@@ -2,13 +2,16 @@
 import glob
 import os
 import sys
-import warnings
 
 import matplotlib
 
 matplotlib.use("WXAgg")
-from gooey import Gooey, GooeyParser
+import argparse
+import datetime
 
+from loguru import logger
```

Comment:
This import does not respect isort. Did you try to run it?

Reply:
Hm, pre-commit runs locally before I commit now, so it ran it before commiting I checked. Also I just re-ran only isort with verbose, but it does not detect any issues. Is there any chance it misses things sometimes?